v1.0.1.md

```diff
@@ -0,0 +1,123 @@
+* Virtual machine status stuck in "Starting" [[#1822](https://github.com/harvester/harvester/issues/1822)]
+* node disk manager relies on unstable devPath to retrieve fiilesystem info [[#1829](https://github.com/harvester/harvester/issues/1829)]
+* unable to add node as VIP:8443/ping is connection refused [[#1830](https://github.com/harvester/harvester/issues/1830)]
+* 1.0.1 opensuse/SLE change of qemu-guest-agent [[#1833](https://github.com/harvester/harvester/issues/1833)]
+* Builder package user data / network data mismatch [[#1878](https://github.com/harvester/harvester/issues/1878)]
+* Panic if the vlan nad config is without labels [[#1886](https://github.com/harvester/harvester/issues/1886)]
+* login with member role, no secret api access [[#1894](https://github.com/harvester/harvester/issues/1894)]
+* Adapt node promotion changes from rancher [[#1920](https://github.com/harvester/harvester/issues/1920)]
+* GUI allows selecting the default data disk as an additional disk [[#1924](https://github.com/harvester/harvester/issues/1924)]
+* Refreshing the Virtual Machine page changes the status from "Running" to "Starting" [[#1925](https://github.com/harvester/harvester/issues/1925)]
+* GUI throws errors after restoring a new VM [[#1928](https://github.com/harvester/harvester/issues/1928)]
+* unable to setup NFS backup-target [[#1952](https://github.com/harvester/harvester/issues/1952)]
+* Unable to provision RKE2 cluster from rancher, stuck in provisioning on bootstrap node [[#1954](https://github.com/harvester/harvester/issues/1954)]
+* "Default version" doesn't work on Modify template page [[#1968](https://github.com/harvester/harvester/issues/1968)]
+* clone volume has an error attach vm [[#1977](https://github.com/harvester/harvester/issues/1977)]
+* stop VM, the state of the VM changes to VM error, pause a vm, an inappropriate message will be displayed in the vm list [[#1987](https://github.com/harvester/harvester/issues/1987)]
+* VM created by VM Template with Access Credentials use the same secret [[#1989](https://github.com/harvester/harvester/issues/1989)]
+* Unable to join a cluster [[#1993](https://github.com/harvester/harvester/issues/1993)]
+* harvester-cloud-provider rely on CNI to communicate with host [[#1999](https://github.com/harvester/harvester/issues/1999)]
+* Harvester UI cloudconfig `packages` option being overwritten on VM creation [[#2006](https://github.com/harvester/harvester/issues/2006)]
+
+# Known Issues
+* Deleting a host holding the VIP in the cluster makes the VIP inaccessible [[#1398](https://github.com/harvester/harvester/issues/1398)]
+* Manual upgrade from 0.3.0 to 1.0.0 [[#1665](https://github.com/harvester/harvester/issues/1665)]
+* When using Load Balancer IPAM `Pool`s, user is only able to provision 1 LB successfully. Subsequent LBs using `Pool` will fail [[#2041](https://github.com/harvester/harvester/issues/2041)]
```

Comment:
Can u please check with David or Lanfo if they can reproduce it? at least I can't reproduce it in my env so might be an env related issue.